blog/_posts/2016-10-06-julia-0.5-highlights.md

```diff
@@ -0,0 +1,726 @@
+---
+layout: post
+title:  Julia 0.5 Highlights
+author: Stefan Karpinski
+---
+
+Julia 0.5 is a pivotal release.
+It introduces more transformative features than any release since the first official version.
+Moreover, several of these features set the stage for even more to come in the [lead up to Julia 1.0](https://www.youtube.com/watch?v=5gXMpbY1kJY).
+In this post, we'll go through some of the major changes in 0.5, including improvements to functional programming, comprehensions, generators, arrays, strings, and more.
+
+## Functions
+
+Julia has always been a functional language in the technical sense: functions are values that can be passed to and from other functions (i.e. [higher-order functions](https://en.wikipedia.org/wiki/Higher-order_function)) and there is full support for [lambdas](https://en.wikipedia.org/wiki/Anonymous_function).
+Before this release, however, anonymous and higher-order functions came with a significant performance cost, and in a language that targets high-performance technical computing, that's a serious limitation.
+So the Julia standard library and ecosystem have been rife with work-arounds to get the expressiveness of functional programming without the performance problems.
+But the right solution, of course, is to make functional programming fast – ideally just as fast as the optimal hand-written version of your code would be.
+In Julia 0.5, it is.
+And that changes everything.
+
+This change is so important that there will be a separate blog post about it in the coming weeks, explaining how higher-order functions and lambdas have been made so efficient, as well as detailing the kinds of zero-cost abstractions these changes enable.
+But for now, I'll just tease with a little timing comparison.
+First, some definitions – they're the same in both 0.4 and 0.5:
+
+    v = rand(10^7);                   # 10 million random numbers
```

Comment:
So from that example it looks like I should put those markers around each block instead of indenting or using backticks?